R/utils.R

```diff
@@ -14,6 +14,10 @@ os_is_macos <- function() {
   isTRUE(Sys.info()[["sysname"]] == "Darwin")
 }
 
+is_verbose_mode <- function() {
+  getOption("cmdstanr_verbose", default = FALSE)
```

Comment:
The way you had this function was totally fine, but to simplify things I changed it to use `getOption()`, which conveniently handles the `NULL` case with the `default` argument.

Reply:
Ah didnt know that. Thanks!